src/ControlSystem/UpdateFunctionOfTime.hpp

```diff
@@ -50,13 +44,7 @@ struct UpdateMultipleFunctionsOfTime {
           f_of_t_list,
       const double update_time,
```

Comment:
Remove consts from declarations.  Also get the `not_null` arguments for all three structs.

Reply:
Could you explain what you mean by "get the `not_null` arguments"? I'm a bit confused what I should be "get"ting

Reply:
Ohh haha gotcha, my bad. I thought you meant like `get<>()` so I was confused.